apps/src/blockly/addons/contextMenu.js

```diff
@@ -49,10 +47,7 @@ const registerBlockPasteFromStorage = function() {
     },
     callback: function(scope) {
       const copyData = localStorage.getItem('blocklyStash');
-      Blockly.Xml.domToBlockSpace(
-        Blockly.mainBlockSpace,
-        Blockly.Xml.textToDom(`<xml>${copyData}</xml>`)
-      );
+      Blockly.mainBlockSpace.pasteBlock_(false, JSON.parse(copyData));
```

Comment:
Curious why you're calling on private pasteBlock_ function here vs public paste? Could use `Blockly.mainBlockSpace.paste(JSON.parse(copyData))`